pyi.py

```diff
@@ -1621,6 +1623,26 @@ def _check_if_expression(self, node: ast.expr) -> None:
         else:
             self.error(node, Y002)
 
+    def _check_for_Y066_violations(self, node: ast.If) -> None:
+        test = node.test
+        if not isinstance(node.test, ast.Compare):
```

Comment:
Should fix the mypy error:

```suggestion
        if not isinstance(test, ast.Compare):
```

Reply:
I was faster ;)